tests/test_https.py

```diff
@@ -48,3 +50,23 @@ def test_run_cache_setup(mocker):
 
     assert saved_env["HTTPS_DOMAIN"] == env["HTTPS_DOMAIN"]
     assert saved_env["HTTPS_ADMIN_EMAIL"] == env["HTTPS_ADMIN_EMAIL"]
+
+def test_support_for_french(mocker):
+    mocker.patch('init-odkx-sync-endpoint.is_enforce_https', return_value=True)
+    # Mock certificate setup during test as a no-op since it's expensive
+    mocker.patch('init-odkx-sync-endpoint.setup_certbot_certificate', return_value=None)
+    mocker.patch('init-odkx-sync-endpoint.setup_manual_certificate', return_value=None)
+
+    jw.setup_translator('fr')
```

Comment:
@jesse5555 this must be changing the inputs to French.  Kindly share @matt9j if you know what i might be missing here